graphsdk/src/androidTest/java/com/microsoft/graph/serializer/CalendarSerializerTests.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.graph.serializer;
+
+import android.test.AndroidTestCase;
+
+import java.util.Calendar;
+
+
+/**
+ * Test cases for {@see CalendarSerializer}
+ */
+public class CalendarSerializerTests extends AndroidTestCase {
+
+    public void testCalendarSerializer() throws Exception {
+        Calendar calendar = Calendar.getInstance();
```

Comment:
Note: we should never use the time from a calendar instance that is returned at runtime, it is too easy to hit a month/day/month/hour/minute boundary that causes intermittent failures.  All calendar data should be fully deterministic